modules/installation-custom-ibm-power-vs.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_power/installing-ibm-power-vs-private-cluster.adoc
+
+:_content-type: CONCEPT
+[id="installation-custom-ibm-power-vs-requirements_{context}"]
+== Requirements for using your VPC
+
+You must correctly configure the existing VPC and its subnets before you install the cluster. The installation program does not create a VPC or VPC subnet in this scenario.
+
+The installation program cannot:
+
+* Subdivide network ranges for the cluster to use
+* Set route tables for the subnets
+* Set VPC options like DHCP
+
+include::snippets/custom-dns-server.adoc[]
+
+[id="installation-custom-ibm-power-vs-validation_{context}"]
+== VPC validation
+
+The VPC and all of the subnets must be in an existing resource group. The cluster is deployed to this resource group.
+
+As part of the installation, specify the following in the `install-config.yaml` file:
+
+* The name of the resource group
+* The name of VPC
+* The name of the VPC subnet
+
+To ensure that the subnets that you provide are suitable, the installation program confirms the following:
+
+* All of the subnets that you specify exist.
+* For each availability zone in the region, you specify:
+** One subnet for control plane machines.
+** One subnet for compute machines.
+* The machine CIDR that you specified contains the subnets for the compute machines and control plane machines.
```

Comment:
These are not true for Power VS, sorry I didn't catch them earlier. You can push the first bullet into the sentence above it and that should be enough.

Reply:
Thanks @mjturek . Have updated the module.